src/pymedphys/dicom/__init__.py

```diff
@@ -29,7 +29,7 @@
 Available Functions
 -------------------
 >>> from pymedphys.dicom import (
-...     anonymise_dicom,
+...     anonymise_dicom_dataset,
```

Comment:
@Centrus007 Up to you if you are happy or not with this name change. Let me know.

Reply:
Yep that's better.

I'm imagining a dicom anonymise cli that calls a `anonymise_dicom_file` with a filepath parameter, which runs `pydicom.dcmread()` on the filepath and calls this.